src/main/flight/mixer.c

```diff
@@ -747,6 +747,11 @@ void applyMotorStop(void)
 
 FAST_CODE_NOINLINE void mixTable(timeUs_t currentTimeUs, uint8_t vbatPidCompensation)
 {
+#ifdef USE_DYN_LPF
+#define DYN_LPF_THROTTLE_STEPS 100
```

Comment:
Since preprocessor macros have no scope, I would either place this one somewhere at the top of this file, or undefine it after the last usage.